### PR TITLE
feat: wallet linking has been implemented

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
This PR implements linking part on wallet itself

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

**Comment by plondon** [comment]
squished wallet logo: 
Screen Shot 2020-08-24 at 11 03 38 AM

when user needs to verify email I see a 0:
Screen Shot 2020-08-24 at 11 07 26 AM

it says Im gold verified but Im not?
Screen Shot 2020-08-24 at 11 09 14 AM

File: packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/template.conflict.tsx [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+type State = {
+  allowSubmit: boolean
+  considerAddress: boolean
+  considerEmail: boolean
+  selectedAddress: string | null
+  selectedEmail: string | null
+}
+
+class Conflict extends PureComponent<
+  Props & { close: () => void; error: any },
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/template.conflict.tsx:54]: Are you able to add an LinkExchangeErrorType by any chance?

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/template.conflict.tsx]: Yup I moved it 👍

File: packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/template.success.tsx [hunk 2]
```diff
@@ -15,50 +23,264 @@ const getIcon = tier => {
...
+
+const ListText = styled(Text)`
+  text-align: left;
+  margin-left: 18px;
+  margin-top: 8px;
+  span {
+    text-indent: -12px;
+    display: inline-block;
+    padding-left: 12px;
+  }
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/template.success.tsx:72]: whole magic is done here we do alignment of text to move every next line to be aligned with text from 1st line where we have dash

File: packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/styles.tsx [hunk 3]
```diff
@@ -58,3 +62,35 @@ export const ButtonWrapper = styled.div`
   width: 100%;
   flex-direction: column;
 `
+
+export const ListWrapper = styled.div`
+  max-width: 356px;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/styles.tsx:67]: does this cause issues on mobile @milan-bc ?

File: packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/styles.tsx [hunk 4]
```diff
@@ -58,3 +62,35 @@ export const ButtonWrapper = styled.div`
...
+  ul {
+    list-style-type: none;
+    /*use padding to move list item from left to right*/
+    padding-left: 1em;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/styles.tsx:71]: px

File: packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/styles.tsx [hunk 5]
```diff
@@ -58,3 +62,35 @@ export const ButtonWrapper = styled.div`
...
+  }
+
+  ul li:before {
+    content: '–';
+    position: absolute;
+    /*change margin to move dash around*/
+    margin-left: -1em;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/styles.tsx:78]: px

File: packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/template.conflict.tsx [hunk 6]
```diff
@@ -50,8 +50,19 @@ type State = {
...
+  address: {
+    exchange: string
+    wallet: string
+  }
+  email: {
+    exchange: string
+    wallet: string
+  }
+}
+
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/template.conflict.tsx:63]: 👏

**Comment by plondon** [comment]
@milan-bc do we need to wait for a BE deployment to production for this to go live?

**Comment by milan-bc** [comment]
@plondon yeah we need BO to be improved, @schnogz is on that one

File: packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/styles.tsx (hunk 4 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/styles.tsx:71]: this is dead code, I'll remove

File: packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/styles.tsx (hunk 3 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/styles.tsx:67]: Nope it works as a charm :)

**Pull request merged**